injector/LogInjector.py

```
@@ -1,181 +1,78 @@
-            injected tree and adds the corresponding log statements. It 
-            then processes the body of the the root node.
-
-            :param rootNode: AST Node which contains body nodes
-            :param injectedTree: AST Module to inject logs into
-            :param isSibling: Indicates if this node is a sibling 
-                node (elif, else,except etc)
-        '''
-        node = NodeExtractor(rootNode)
-        sstRootNode = self.sst.addAstNode("root", node, isSibling)
-        self.sst.activeNode = sstRootNode
+from injector import helper
```

Ruff (0.8.2)

`injector.helper` imported but unused

Remove unused import: `injector.helper`

(F401)